google-beta/resource_compute_instance_group_manager.go

```diff
@@ -14,6 +15,11 @@ import (
 	compute "google.golang.org/api/compute/v1"
 )
 
+var (
+	instanceGroupManagerIdRegex     = regexp.MustCompile("^" + ProjectRegex + "/[a-z0-9-]+/[a-z0-9-]+$")
+	instanceGroupManagerIdNameRegex = regexp.MustCompile("^[a-z0-9-]+$")
```

Comment:
Does it make sense to keep the trailing hyphen match here? Is "ab01-" a valid IdName?

Reply:
I don't believe it's a valid name, I just also don't know that it's worth catching that at this level. The point of the regexes is to validate that they supplied a project/zone/name string, not really to validate that that string constitutes a valid project, zone, or name.